dashboard/app/models/pd/workshop.rb

```diff
@@ -481,7 +481,7 @@ def workshop_dashboard_url
   end
 
   def associated_online_course
-    Plc::Course.find_by(name: WORKSHOP_COURSE_ONLINE_LEARNING_MAPPING[course]) if WORKSHOP_COURSE_ONLINE_LEARNING_MAPPING[course]
+    ::Course.find_by(name: WORKSHOP_COURSE_ONLINE_LEARNING_MAPPING[course]).try(:plc_course) if WORKSHOP_COURSE_ONLINE_LEARNING_MAPPING[course]
```

Comment:
`WORKSHOP_COURSE_ONLINE_LEARNING_MAPPING[course]` gives us a string. Just because that string exists, doesn't mean we have a Course of that name.